### DISCUSSION TITLE
How can I set a new main page?

### DISCUSSION THREAD

**Discussion by derJuba007** [automadcms/discussions#15] [Q&A]
Hi, I recently experimented a little with version 2 of Automad and I'm really excited about this CMS.

I'm just stuck on one little thing at the moment. I can't manage to create a new main page. As I understand the system of the pages, there is a main page at the beginning, which is also the homepage (welcome). This has 3 subpages.

If I now create a new page, I only have the option of setting it as a subpage of the existing Welcome page.

It is also not possible for me to set the page as the main page using drag and drop in the dashboard sidebar.

Is this a bug or am I missing something? I haven't found anything about this in the documentation.

**Comment by marcantondahmen** [comment]
There can only be one root page. It is similar to the document root on a static site. It is the root of your site tree. You can edit the content and its settings but it cannot be moved. It behaves the same like a root directory on a file system.

**Comment by derJuba007** [comment]
Oh okay, because it looks like this on the automad website.

This is what I want to achieve:
<img width="113" alt="Screenshot 2024-09-02 194513" src="https://github.com/user-attachments/assets/dfbf4524-b0b1-46fe-85e2-e0ff1650975e">

This is what I get:
<img width="119" alt="Screenshot 2024-09-02 194628" src="https://github.com/user-attachments/assets/afb5d096-c934-4e13-a3c4-2407141bccd5">

**Comment by marcantondahmen** [comment]
You can create the same structure. Just move a page inside another one. Pages are just directories.

However, I would recommend to directly use v2 for a new project.
